format/swf/Writer.hx

```diff
@@ -60,8 +60,23 @@ class Writer {
 		writeEnd();
 	}
 
+	function signedMinBits(values: Array<Int>): Int {
+		var x: Int = 0;

```

Comment:
This method should be added to `Tools`. Also I think it'd be clearer to return 0 if `Tools.minBits` returns 0.